source/group_eq_bot/interfaces/telegram_event_handlers/messages/command/handler.py

```diff
@@ -0,0 +1,51 @@
+from dataclasses import dataclass
+from telegram.ext import CommandHandler, ContextTypes
+
+from telegram import Update as TelegramEvent
+
+from interfaces.telegram_event_router.router import route_event
+from utilities.internal_logger.logger import logger
+
+
+@dataclass
+class StartCommandHandler:
+    CONTEXT_DEFAULT_TYPE = ContextTypes.DEFAULT_TYPE
+
+    command_to_be_written = 'start'
+    callback_to_call = None
+
+    handler = CommandHandler(command=command_to_be_written,
+                             callback=callback_to_call)
+
+    def start(self, event: TelegramEvent, context: CONTEXT_DEFAULT_TYPE) -> int:
+        """ Function, which handles `start` command update from CommandHandler. """
+
+        username = event.message.from_user
+        logger.info(f'User {username.full_name} started the conversation with bot')
+        logger.info(event)
+
+        await event.message.reply_text(text=first_question)
+
+        return FIRST_ANSWER
+
+    def handle(self) -> 'StartCommandHandler':
+        self.callback_to_call = self.start()
+        return self
+
+
+@dataclass
+class EndCommandHandler:
+    command_to_be_written = 'end'
+    callback_to_call = route_event
+
+    handler = CommandHandler(command=command_to_be_written,
+                             callback=callback_to_call)
```

Comment:
1. suggest to rename class as `End`, because the path to the file is self-explanatory which functionality you are interacting with
2. move class to it's own file, which can be named as `end.py`

The command directory tree structure can be done as:
```
- telegram_event_handlers
    - command
        - name
            - end.py
            - start.py
            - help.py
        - handler.py
```

Reply:
deleted general bot commands